sql/moz-fx-data-shared-prod/org_mozilla_fenix/baseline_clients_daily/metadata.yaml

```diff
@@ -1,12 +0,0 @@
----
-# Generated via bigquery_etl.glean_usage
```

Comment:
These files should get created as part of the SQL generation afaik, so no need to check them in to `main`